packages/utils/src/lib/reports/generate-md-report-categoy-section.ts

```diff
@@ -0,0 +1,148 @@
+      })),
+      alignment: reportOverviewTableAlignment,
+    };
+    return tableSection(tableContent);
+  }
+  return '';
+}
+
+export function categoriesDetailsSection(
+  report: Pick<ScoredReport, 'categories' | 'plugins'>,
+): string {
+  const { categories, plugins } = report;
+
+  const categoryDetails = categories.flatMap(category => {
+    const categoryTitle = h3(category.title);
+    const categoryScore = `${scoreMarker(
+      category.score,
+    )}${SPACE}Score:  ${style(formatReportScore(category.score))}`;
+
+    const categoryMDItems = category.refs.map(ref => {
+      // Add group details
+      if (ref.type === 'group') {
+        const group = getSortableGroupByRef(ref, plugins);
+        const groupAudits = group.refs.map(groupRef =>
+          getSortableAuditByRef(
+            { ...groupRef, plugin: group.plugin, type: 'audit' },
+            plugins,
+          ),
+        );
+        const pluginTitle = getPluginNameFromSlug(ref.plugin, plugins);
+        return categoryGroupItem(group, groupAudits, pluginTitle);
+      }
+      // Add audit details
+      else {
+        const audit = getSortableAuditByRef(ref, plugins);
+        const pluginTitle = getPluginNameFromSlug(ref.plugin, plugins);
+        return categoryRef(audit, pluginTitle);
+      }
+    });
+
+    return section(
+      categoryTitle,
+      metaDescription(category),
+      categoryScore,
+      ...categoryMDItems,
+    );
+  });
+
+  return lines(h2(CATEGORIES_TITLE), ...categoryDetails);
+}
+
+export function categoryRef(
+  { title, score, value, displayValue }: AuditReport,
+  pluginTitle: string,
+): string {
+  const auditTitleAsLink = link(
+    `#${slugify(title)}-${slugify(pluginTitle)}`,
+    title,
+  );
+  const marker = scoreMarker(score, 'square');
+  return li(
+    `${marker}${SPACE}${auditTitleAsLink}${SPACE}(_${pluginTitle}_) - ${style(
+      (displayValue || value).toString(),
+    )}`,
+  );
+}
+
+export function categoryGroupItem(
+  { score = 0, title }: ScoredGroup,
+  groupAudits: AuditReport[],
+  pluginTitle: string,
+): string {
+  const groupTitle = li(
+    `${scoreMarker(score)}${SPACE}${title}${SPACE}(_${pluginTitle}_)`,
+  );
+  const auditTitles = groupAudits.map(
+    ({ title: auditTitle, score: auditScore, value, displayValue }) => {
+      const auditTitleLink = link(
+        `#${slugify(auditTitle)}-${slugify(pluginTitle)}`,
+        auditTitle,
+      );
+      const marker = scoreMarker(auditScore, 'square');
+      return indentation(
+        li(
+          `${marker}${SPACE}${auditTitleLink} - ${style(
+            String(displayValue ?? value),
+          )}`,
+        ),
+      );
+    },
+  );
+
+  return lines(groupTitle, ...auditTitles);
+}
+
+export function auditDetailsAuditValue({
+  score,
+  value,
+  displayValue,
+}: AuditReport) {
+  return `${scoreMarker(score, 'square')} ${htmlFontStyle(
+    String(displayValue ?? value),
+  )} (score: ${formatReportScore(score)})`;
+}
```

Comment:
This function is duplicated in `generate-md-report.ts`. Please delete one of them.

Reply:
removed in category section